server/coordinator_test.go

```diff
@@ -315,36 +315,40 @@ func MaxUint64(nums ...uint64) uint64 {
 	return result
 }
 
-func (s *testCoordinatorSuite) TestCheckRegion(c *C) {
-	_, opt, err := newTestScheduleConfig()
-	c.Assert(err, IsNil)
+func (s *testCoordinatorSuite) prepare(cfg *config.ScheduleConfig, opt *config.ScheduleOption, c *C) (*testCluster, *heartbeatStreams, func(), *coordinator) {
```

Comment:
It's better to use it for other tests in the file?

Reply:
Good point. I use some and find there are many changes, I will push them in new PR.